cpp/src/arrow/compute/kernels/aggregate_mode.cc

```diff
@@ -28,6 +29,9 @@ namespace internal {
 
 namespace {
 
+constexpr char kModeFieldName[] = "modes";
+constexpr char kCountFieldName[] = "counts";
```

Comment:
Can you keep "mode", "count" as documented?

Reply:
Done